### DISCUSSION TITLE
台積電赴美搶劫論瘋傳！

### DISCUSSION THREAD

**Discussion by moonnight3503** [1101-GS4719-C/Python-Community-GS4719-C#893] [General]
台灣的護國神山台積電在美國時間7日時，舉辦亞利桑那州廠的移機典禮，連美國總統拜登都親自出席，卻也導致網路上出現一股「美國搶劫」、「技術外流」等輿論瘋傳，卻有專家質疑「看起來比較像是台積電搶劫美國人吧！」。我認為，科技業往國外發展是好選擇，畢竟台灣太小，能夠去到國外與他國技術交流必定會對科技業的發展及演進有所幫助！但應該也不會說到搶劫美國人啦哈哈。

https://tw.news.yahoo.com/%E5%8F%B0%E7%A9%8D%E9%9B%BB%E8%B5%B4%E7%BE%8E%E6%90%B6%E5%8A%AB%E8%AB%96%E7%98%8B%E5%82%B3-%E5%B0%88%E5%AE%B6%E7%9C%8B%E6%B3%95%E6%9B%9D-005248635.html

**Comment by HuangHsinTzu** [comment]
我們的護國神山去國外設廠
民眾難免會有遐想，會擔心
但我個人認為台積電是不可能整個搬去美國的
畢竟要做出這麼精細的晶片
還是需要願意為五斗米折腰的台灣人啊
(純屬個人見解XD)

**Comment by Chiway1229** [comment]
我是覺得人們不用擔心，反正去美國設廠能提升雙方人才交流的機會的話也是不錯啦 哈哈

**Comment by bonny123865** [comment]
產品是made in USA或是in Taiwan的知名度可能會差很多
但是我覺得到美國設廠也可以精進技術，更接近美國地區的供應市場
也不完全是壞事!

**Reply by wangtyphoon** [reply]
股權超過一半是外資能說是台灣公司嗎?

**Comment by WEI-CHEN-CHUN** [comment]
過度遐想導致人心惶惶，一間技術領先全球的台灣企業，到國外設廠，其實蠻正常的。

**Comment by yuxinlin93** [comment]
這個新聞真的很大，甚至還有新聞台邀請財金專家講解，還關係到軍事甚麼的，超好笑的。但我個人認為這個廠在美國營運應該會有很高的困難度，因為台灣人的工作習慣跟美國人的工作習慣很不同，如果初期用在台灣的管理模式管理美國員工，絕對是沒辦法馬上套入的，所以應該會需要有一段時間調整管理模式，可不可行一切還都是未知數。

**Comment by evanchou1204** [comment]
只要老闆還是台灣人，我相信台積電不會做出背棄祖國的事情的，畢竟體內留著台灣人的血。

**Comment by CPCpyt** [comment]
我覺得這是很好的進展，
不過人說台灣人薪資便宜而且工作認真老實，
我想台積電不會輕易放棄台灣的！
